React/DevSupport/RCTInspectorDevServerHelper.mm

```diff
@@ -44,6 +48,10 @@
 static NSURL *getAttachDeviceUrl(NSURL *bundleURL, NSString *title)
 {
   NSNumber *metroBundlerPort = @8081;
+  NSString *metroBundlerPortStr = [[[NSProcessInfo processInfo] environment] objectForKey:@"RCT_METRO_PORT"];
+  if (!metroBundlerPortStr) {
```

Comment:
This doesn't look correct to me?

Also, should we make sure that the string is non-empty? If it is `""`, then we don't want to apply the string.

Reply:
Ah, you're right.  I'll fix that up.